content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -275,74 +279,94 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
                 </h1>
               </div>
               <span>
-                <NextLink href={`${guideTypeUrl}#${stopNumber}`}>
+                <TransitionLink
+                  href={`${guideTypeUrl}#${currentStop.number}`}
+                  onFocus={() =>
+                    setViewTransitionName(`player-${currentStop.number}`)
+                  }
+                  onMouseEnter={() =>
+                    setViewTransitionName(`player-${currentStop.number}`)
+                  }
```

Comment:
This could be moved to TransitionLink itself and you pass in an `id` instead, which it takes in. (So I guess the `useState` moves there as well)

Reply:
I think because there can be any number of `view-transition-name`s on a page that might or might not need to change, and the reason they might have to change wouldn't necessarily be due to hovering/focusing a link, I don't feel like moving this logic into the `TransisitonLink` component is the right way to go. I think it's job should just be to run the `startViewTransition` callback.